src/components/AppProvider/utilities/createPolarisContext/createPolarisContext.ts

```diff
@@ -29,7 +29,7 @@ export default function createPolarisContext(
   let themeContext: CreateThemeContext | undefined;
   if (contextOne && 'logo' in contextOne) {
     themeContext = contextOne as CreateThemeContext;
-    appProviderContext = contextTwo;
+    appProviderContext = contextTwo as CreateAppProviderContext;
```

Comment:
`CreateThemeContext` and `CreateAppProviderContext ` don't have as much overlap anymore